### PR TITLE
Fix problems with automated test

### PR DESCRIPTION
**Pull request by swedchef**
Restructured repo so main py is in app-folder.
Created __init__.py files and commented out unused items from test
file. Modified workflow to find all requirements.txt files and
install them.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 RUN pip3 install --upgrade pip && pip3 install -r requirements.txt --no-cache-dir && rm requirements.txt
 
-# Copy required files into container
-COPY filemover.py /
+# LAYER 3 / Copy required files into container
+COPY app/filemover.py /
 
-# Run the application
+# LAYER 4 / Run the application
 CMD ["python3", "-u", "filemover.py"]
```
Comment (stigsberg) [inline Dockerfile:12]:
```suggestion
CMD ["python3", "-u", "app/filemover.py"]
```

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,12 @@
-# Select base-image
+# LAYER 1 / Select base-image
 FROM python:3.10.0-slim-bullseye
 
-# Load python requirements, install them and clean up.
-COPY requirements.txt /
+# LAYER 2 / Load python requirements, install them and clean up.
+COPY app/requirements.txt /
 RUN pip3 install --upgrade pip && pip3 install -r requirements.txt --no-cache-dir && rm requirements.txt
```
Comment (stigsberg) [inline Dockerfile:6]:
```suggestion
RUN pip3 install --upgrade pip && pip3 install -r app/requirements.txt --no-cache-dir && rm app/requirements.txt
```

**Review by stigsberg — CHANGES_REQUESTED** [review]
Suggested change to the dockerfile

File: Dockerfile (hunk 1 above)
Comment (swedchef) [reply Dockerfile:12]: That will break the container. As you can see in line 9, I take the file from the app-folder, but move it into the root directory. If line 9 was "copy app/filemover.py /app/", you would have been correct though.

**Review by stigsberg — APPROVED** [review]
LGTM

File: Dockerfile (hunk 1 above)
Comment (swedchef) [reply Dockerfile:12]: That will break the container. As you can see in line 9, I take the file from the app-folder, but move it into the root directory. If line 9 was "copy app/filemover.py /app/", you would have been correct though.

**Pull request merged**